__tests__/apiTests.test.js

```diff
@@ -88,4 +88,25 @@ describe("GET /api/articles/:article_id",()=>{
 
         })
     })
+})
+
+describe("GET /api/articles",()=>{
+    test("get status code 200, and all the articles in an array with comment_count and ordered by date descending",()=>{
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+            .then(({body})=>{
+                const articles=body.articles
+                expect(typeof articles).toBe("object")
+                expect(articles).toBeSortedBy("created_at",{descending: true})
+                articles.forEach((article)=>{
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)